.circleci/fundingcirclebot.key

```diff
@@ -0,0 +1,60 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
Can you explain why it's okay to place this under version control?

Reply:
This is just an unpublished key for demo purposes. We should make a real one and encrypt it.

Reply:
I updated the PR with an encrypted version of a new private key. We'll need to publish this and store the private key somewhere safe.